pumpkin-config/src/lib.rs

```diff
@@ -159,7 +160,10 @@ trait LoadConfiguration {
 
 impl LoadConfiguration for AdvancedConfiguration {
     fn get_path() -> &'static Path {
-        Path::new("features.toml")
+        match env::var("FEATURES") {
```

Comment:
Personally I'd like to have this env var be more descriptive and as such less likely to cause a name collision with out other environment variables.

Possible suggestion would be to change it to `FEATURES_PATH`.

Reply:
Renamed.